test/k6/src/tests/orders-org-no-v2.js

```
@@ -0,0 +1,175 @@
+import * as setupToken from "../setup.js";
+import * as ordersApi from "../api/notifications/v2.js";
+import { post_email_order_v2, get_email_shipment, post_sms_order_v2, get_sms_shipment, setEmptyThresholds } from "./threshold-labels.js";
+import { getShipmentStatus } from "./orders-v2.js";
+import { scopes, resourceId, options } from "../shared/variables.js";
+import { getOrgNoRecipient } from "../shared/functions.js";
+
+const orderRequestJson = JSON.parse(
+    open("../data/orders/order-v2-org.json")
+);
+
+export const options = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'options'. Consider to delete it or rename it.

'options' is defined here:


(lint/suspicious/noRedeclare)